download-robolectric-deps.sh

```diff
@@ -22,3 +22,4 @@ mkdir -p maps/src/test/resources && cp robolectric-deps.properties "$_"
 mkdir -p errors/src/test/resources && cp robolectric-deps.properties "$_"
 mkdir -p selfie-camera/src/test/resources && cp robolectric-deps.properties "$_"
 mkdir -p qr-code/src/test/resources && cp robolectric-deps.properties "$_"
+mkdir -p draw/src/test/resources && cp robolectric-deps.properties "$_"
```

Comment:
There's a note in the `README` about it:

> #### `Unable to resolve artifact: Missing` while running tests
>
> This is encountered when Robolectric has problems downloading the jars it needs for different Android SDK levels. If you keep running into this you can download the JARs locally and point Robolectric to them by doing:
> ```
> ./download-robolectric-deps.sh
> ```

Basically it allows all the Robolectric jars to be downloaded in one go rather than having one randomly fail to download half way through a test run (which is annoyingly common) and prevents you from running into a scenario where you're offline and never ran a test that needs a particular jar (ones that configure particular SDK levels). We should have it for all the modules that use Robolectric (although we might be missing one or two).